man/rshim.8

```diff
@@ -111,18 +111,26 @@ Run in forground.
 
 -F, --force
 .in +4n
-Put rshim driver in ownership-forceable mode (aka. force mode). In this mode, the driver will send
-a one time ownership request command to the other rshim backends upon start-up
-if the rshim interface is already attached by another backend.  It also enables
-"/dev/rshim<N>/" creation even when rshim is not attached, allowing user to
-manually send a ownership request (aka FORCE_CMD) to "/dev/rshim<N>/misc"
-interface at any time.
+Put rshim driver in ownership-forceable mode (aka. force
```

Comment:
This line seems short. Is it on purpose?

Reply:
Good catch. Will change.